warehouse/packaging/models.py

```diff
@@ -717,7 +718,12 @@ def uploaded_via_trusted_publisher(self) -> bool:
         """Return True if the file was uploaded via a trusted publisher."""
         return (
             self.events.where(
-                self.Event.additional.op("->>")("publisher_url").is_not(None)  # type: ignore[attr-defined] # noqa E501
+                or_(
+                    self.Event.additional[  # type: ignore[attr-defined]
+                        "uploaded_via_trusted_publisher"
+                    ].as_boolean(),  # noqa E501
```

Comment:
What happened here with E501? Wouldn't that have remained on a single line like so?

```suggestion
                    self.Event.additional["uploaded_via_trusted_publisher"].as_boolean(),  # type: ignore[attr-defined] # noqa E501
```

Reply:
Yeah, I believe it's now non-operative (since the line is below the `E501` limit). I *think* I can just remove the `noqa` entirely, unless we'd prefer to intentionally push the line over the lint limit?

Reply:
I dropped the E501s here by reflowing.